LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
`LICENSE` is missing a file format i think. perhaps a `.md` or `.txt` would be helpful